profiles/package.deprecated

```diff
@@ -17,6 +17,45 @@
 
 #--- END OF EXAMPLES ---
 
+# Andreas Sturmlechner <asturm@gentoo.org> (2025-08-19)
+# Qt5 upstream OSS support ended on 2020-12-08. Public availability of
+# commercial 5.15 LTS releases is delayed by 1 year since then. The final
+# release happened on 2025-05-19, which means a) we will not get it until
+# 2026-06-19, b) any bugs discovered since last May will not be fixed and
+# c) no bugs since 2020-12-08 that haven't also been affecting Qt6 were
+# fixed at all. Bug 948836
```

Comment:
Think should also add dev-python/pyqt5 to warn about that common indirect dependency, probably lot of consumers of that one in overlays too and warning may help reduce the fallout later.